src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -336,8 +356,7 @@ def run_test(
         super().run_test(
             inv_always_validation_sequence_set,
             inv_eventually_validation_sequence_set,
-            test_timeout_s,
-            tick_duration_s,
+            **kwargs,
```

Comment:
add docs for this

Reply:
what do you mean exactly? the super().run_test function has docs for which arguments it accepts, so do you mean add a comment about the kwargs we're passing in here?